lib/puppet/type.rb

```diff
@@ -116,6 +116,11 @@ def is_capability?
     end
   end
 
+  # Normal types aren't applications
```

Comment:
Describe what it does instead. The documentation makes no sense without reading the implementation.